LICENSE.md

```diff
@@ -0,0 +1,194 @@
+Apache License
```

Comment:
is license file rename really required for the R CMD check to pass? I've seen it's just a line in `.Rbuildignore`.

Reply:
@hadley i'd be happy to merge right away if we don't change `LICENSE` file and just add it to `.Rbuildignore` :)

Reply:
Reverted 😄